client/src/components/UrgencyBadge/UrgencyBadge.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './UrgencyBadge.scss';
+
+interface UrgencyBadgeProps {
+    label: string;
```

Comment:
Consider using literal union type such as label: "medium" | "low" | "high" to make sure we pass only accepted values